icu4c/source/test/intltest/numbertest_api.cpp

```diff
@@ -3180,6 +3180,78 @@ void NumberFormatterApiTest::roundingOther() {
             u"0.000",
             u"0.000");
 
+    assertFormatDescending(
```

Comment:
could you also add a test case of what anba put into https://unicode-org.atlassian.net/browse/ICU-21668 here so we can show the code fix truly fix that exact case?

```
auto increment = NumberFormatter::with()
      .precision(Precision::increment(5000).withMinFraction(0))
      .roundingMode(UNUM_ROUND_UP)
      .locale(Locale::getEnglish())
      .formatDouble(5.625, status)
      .toString(status);

```

Reply:
waiting for that particular exact test case to be added.

Reply:
waiting that exact test case to be added

Reply:
added.